prow/cluster/jobs/istio/operator/istio.operator.master.yaml

```diff
@@ -36,6 +36,33 @@ presubmits:
       nodeSelector:
         testing: test-pool
 
+  - name: operator-build-master
+    <<: *job_template
+    context: prow/build.sh
+    always_run: true
+  # TODO: make required once stable
+    optional: true
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/build.sh
```

Comment:
which node pool do you want to use for this? build-pool or test-pool?

Reply:
Not sure what the delta is?

Reply:
Not sure what the delta is?

Reply:
Not sure what the delta is?

Reply:
what I mean is what is the difference between the build-pool and test-pool?

Cheers
-steve

Reply:
At the moment test-pool has more nodes 20 vs build-pool 10.

Reply:
is the prow rebase on top of master prow done?

Cheers
-steve

Reply:
work in progress

Reply:
yes, done